packages/shared/routes/setup/Congratulations.svelte

```diff
@@ -109,11 +67,80 @@
             } else {
                 _exportMigrationLog()
             }
-        } else {
-            $appRouter.next()
         }
     }
 
+    const migrateAnotherProfile = (): void => {
```

Comment:
after fixing the conflicts, this is where we need to add the new code to reset the migration data
```
    const migrateAnotherProfile = (): void => {
        cleanupSignup()
        $appRouter.reset()
    }
```